docs/documentation.html

```diff
@@ -194,8 +197,15 @@
     <h2><a id="connect" href="#connect">8. Kafka Connect</a></h2>
     <!--#include virtual="connect.html" -->
 
-    <h2><a id="streams" href="#streams">9. Kafka Streams</a></h2>
-    <!--#include virtual="streams.html" -->
+    <h2><a id="streams" href="/documentation/streams">9. Kafka Streams</a></h2>
+    <p>
+        Kafka Streams is a client library for processing and analyzing data stored in Kafka and either write the resulting data back to Kafka or send the final output to an external system. It builds upon important stream processing concepts such as properly distinguishing between event time and processing time, windowing support, and simple yet efficient management of application state.
+    </p>
+    <p>
+        Kafka Streams has a <b>low barrier to entry</b>: You can quickly write and run a small-scale proof-of-concept on a single machine; and you only need to run additional instances of your application on multiple machines to scale up to high-volume production workloads. Kafka Streams transparently handles the load balancing of multiple instances of the same application by leveraging Kafka's parallelism model.
+    </p>
 
-<!--#include virtual="../includes/footer.html" -->
+    <p>Learn More about Kafka Streams read <a href="/documentation/streams">this</a> Section.</p>
```

Comment:
Just to clarify how this works. In the doc directory you give it a "context" object for all the docs - this lives in js/templateData.js. When we render a page from the docs we apply the context to the page using Handlebars.js and that's what lets you do stuff like `{{version}}`.